packages/test-helpers/lib/web3.js

```diff
@@ -0,0 +1,134 @@
+module.exports = (web3) => {
```

Comment:
I ended up not including most of these in https://github.com/aragon/contract-helpers/pull/43.

I'm open to adding them—let's confirm a use case first!